package/MDAnalysis/analysis/align.py

```diff
@@ -332,6 +393,9 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
        Uses :func:`get_matching_atoms` to work with incomplete selections
        and new *strict* keyword. The new default is to be lenient whereas
        the old behavior was the equivalent of *strict* = ``True``.
+
+    .. versionchanged:: 0.15.1
+        Uses :func:`_fit_to` to get new minimum rmsd
```

Comment:
internal change doesn't need to be communicated to the user since it has no visible affect for him. This can be removed.